lib/rspec-virtus/matcher.rb

```diff
@@ -44,7 +44,7 @@ def attribute_type
       end
 
       def attribute_exists?
-        attribute != nil
+        not attribute.nil?
```

Comment:
Could you change this to `!attribute.nil?` please? I've never really been a fan of Ruby's not keyword.

Will merge and release a release a new version once this is done :+1: